tls/s2n_config.c

```diff
@@ -439,7 +438,7 @@ int s2n_config_set_status_request_type(struct s2n_config *config, s2n_status_req
     S2N_ERROR_IF(type == S2N_STATUS_REQUEST_OCSP && !s2n_x509_ocsp_stapling_supported(), S2N_ERR_OCSP_NOT_SUPPORTED);
 
     POSIX_ENSURE_REF(config);
-    config->status_request_type = type;
+    config->ocsp_status_requested_by_user = type == S2N_STATUS_REQUEST_OCSP;
 
```

Comment:
Nit: I generally think this is clearer with parentheses. I worry the mix of "="s will confuse people :)
```suggestion
    config->ocsp_status_requested_by_user = (type == S2N_STATUS_REQUEST_OCSP);

```

Reply:
I agree, it looks better with the parentheses!